Healtheducat/HealthEducat-backend
# Database setup

#### What does this PR do?

- add database structure 

#### Description of Task to be completed?
- every document should have the schema 

#### How should this be manually tested?

- navigate into the project model and check out the models' folder where all documents schemas reside.

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?

#### Screenshots (if appropriate)

#### Questions:
